tests/ci/docker_images/linux-x86/pull_github_pkg.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Note: this file is reserved for pulling Docker images from 'docker.pkg.github.com', which needs external credential.
+# Pull Docker images used by https://github.com/awslabs/aws-c-cal
+# These Docker images are hosted in 'docker.pkg.github.com', which needs GitHub personal access token.
+cur_dir="$(cd "$(dirname "$0")" && pwd)"
+# shellcheck source=./common.sh
```

Comment:
What are these `shellcheck` comments for?

Reply:
https://github.com/koalaman/shellcheck
For bash scripts, some IDEs use shellcheck, which is a static analysis tool. `# shellcheck ` is used to suppress the tool warning because the tool cannot check the runtime value (`cur_dir`).